### ISSUE TITLE
Implement a bilinear initializer for transposed convolution.

### ISSUE THREAD

**Issue by qingqing01** [PaddlePaddle/Paddle#11403]
This is a common use case in  transposed convolution to act as upsampling.

    factor = 2
    w_attr = ParamAttr(learning_rate=0., regularizer=L2Decay(0.),
                      initializer=Bilinear())
    conv_up = fluid.layers.conv2d_transpose(
         input,
          num_filters=C,
          output_size=None,
          filter_size=2 * factor - factor % 2,
          padding=ceil((factor - 1) / 2.),
          stride=factor,
          groups=C,
          param_attr=w_attr,
          bias_attr=False)
